sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -199,7 +201,13 @@ private void setDist(final @NotNull SentryBaseEvent event, final @NotNull String
 
   private void setAppExtras(final @NotNull App app, final @NotNull Hint hint) {
     app.setAppName(ContextUtils.getApplicationName(context, options.getLogger()));
-    app.setAppStartTime(DateUtils.toUtilDate(AppStartState.getInstance().getAppStartTime()));
+    final @NotNull TimeSpan appStartTimeSpan =
+        options.isEnablePerformanceV2()
```

Comment:
Same here, probalby should check for API version 24 together with enablePerformanceV2